tests/integration/test_installation.py

```diff
@@ -16,37 +14,7 @@
 from databricks.labs.ucx.workspace_access.groups import GroupManager
 
 logger = logging.getLogger(__name__)
-
-
-def test_destroying_non_existing_schema_fails_with_correct_message(ws, sql_backend, env_or_skip, make_random):
-    default_cluster_id = env_or_skip("TEST_DEFAULT_CLUSTER_ID")
-    tacl_cluster_id = env_or_skip("TEST_LEGACY_TABLE_ACL_CLUSTER_ID")
-    ws.clusters.ensure_cluster_is_running(default_cluster_id)
-    ws.clusters.ensure_cluster_is_running(tacl_cluster_id)
-
-    backup_group_prefix = "db-temp-"
-    inventory_database = f"ucx_{make_random(4)}"
-    install = WorkspaceInstaller.run_for_config(
-        ws,
-        WorkspaceConfig(
-            inventory_database=inventory_database,
-            groups=GroupsConfig(
-                backup_group_prefix=backup_group_prefix,
-                auto=True,
-            ),
-            log_level="DEBUG",
-        ),
-        prefix=make_random(4),
-        override_clusters={
-            "main": default_cluster_id,
-            "tacl": tacl_cluster_id,
-        },
-    )
-
-    with pytest.raises(OperationFailed) as failure:
-        install.run_workflow("099-destroy-schema")
-
-    assert "cannot be found" in str(failure.value)
+TIMEOUT_VALUE = 5
```

Comment:
It means that tests may run longer in total. Why is it justified? No comment is seen here

Reply:
The integration failures point to a timeout:
```
=========================== short test summary info ============================
FAILED tests/integration/test_installation.py::test_destroying_non_existing_schema_fails_with_correct_message - Failed: DID NOT RAISE <class 'databricks.sdk.errors.OperationFailed'>
FAILED tests/integration/test_installation.py::test_logs_are_available - Failed: DID NOT RAISE <class 'databricks.sdk.errors.OperationFailed'>
FAILED tests/integration/test_installation.py::test_jobs_with_no_inventory_database - AssertionError: assert 1 == 0
 +  where 1 = len(**[TimeoutError('Timed out after 0:02:00')]**)
============= 3 failed, 44 passed, 6 skipped in 2378.96s (0:39:38) =============
Error: Process completed with exit code 1.
```
This is the code
```
        _, errors = Threads.gather("validating results", [validate_tacl, validate_permissions, validate_groups])
        assert len(errors) == 0
```

The timeout fails the integration test. Not clear this is deterministic, 5 minutes may not be enough.